app/src/main/java/com/nononsenseapps/feeder/model/opml/OpmlPullParser.kt

```diff
@@ -263,16 +265,20 @@ class OpmlPullParser(private val opmlToDb: OPMLParserHandler) {
                                 ?.toBoolean()
                                 ?: feed.notify,
                         fullTextByDefault =
-                            (
-                                parser.getAttributeValue(
-                                    OPML_FEEDER_NAMESPACE,
-                                    ATTR_FULL_TEXT_BY_DEFAULT,
-                                )
-                                    ?.toBoolean()
-                                    // Support Flym's value for this
-                                    ?: parser.getAttributeValue(null, ATTR_FLYM_RETRIEVE_FULL_TEXT)
+                            if (parsedUrlOrUri.isNostrUri()) {
+                                false
+                            } else {
```

Comment:
```suggestion
```

Remove this. You should respect the values in the file

Reply:
Please @spacecowboy  actually for Nostr sources, the full text option is not needed because the full article is fetched regardless, and I have not seen how to adapt that option to Nostr sources.